src/cython/cython/cubical_complex.pyx

```diff
@@ -5,13 +5,15 @@ from libcpp.string cimport string
 from libcpp cimport bool
 import os
 
+from numpy import array as np_array
+
 """This file is part of the Gudhi Library. The Gudhi library
    (Geometric Understanding in Higher Dimensions) is a generic C++
    library for computational topology.
 
    Author(s):       Vincent Rouvreau
 
-   Copyright (C) 2016 Inria
+   Copyright (C) 2019 Inria
```

Comment:
I often see copyrights of the form 2016-2019 instead of just the latest year. I doubt that matters though.